fdbserver/Status.actor.cpp

```diff
@@ -2346,11 +2379,16 @@ ACTOR Future<StatusReply> clusterGetStatus(
 				statusObj["fault_tolerance"] = faultToleranceStatusFetcher(configuration.get(), coordinators, workers, extraTlogEligibleZones, minReplicasRemaining, loadResult.present() && loadResult.get().healthyZone.present());
 			}
 
-			JsonBuilderObject configObj = configurationFetcher(configuration, coordinators, &status_incomplete_reasons);
+			state JsonBuilderObject configObj =
+			    configurationFetcher(configuration, coordinators, &status_incomplete_reasons);
 
 			// configArr could be empty
-			if (!configObj.empty())
+			if (!configObj.empty()) {
+				if (primaryDCO.present()) {
```

Comment:
I guess technically we no longer need to hide this field if the configuration is absent, as we don't get this value from the configuration. Let's move it up outside the `!configObj.empty()` if statement.

Reply:
Good catch.